ECMA2Yaml/ECMAHelper/ECMALoader.docs.cs

```diff
@@ -246,11 +246,26 @@ private List<RelatedTag> LoadRelated(List<XElement> relatedElements)
         private TypedContent GetTypedContent(XElement ele)
```

Comment:
we should pass in file name in this function so that we can report the file name when logging user warning